src/bootstrap/bin/rustc.rs

```diff
@@ -306,14 +306,15 @@ fn main() {
     }
 
     // This is required for internal lints.
-    cmd.arg("-Zunstable-options");
     if let Some(crate_name) = args.windows(2).find(|a| &*a[0] == "--crate-name") {
         let crate_name = crate_name[1].to_string_lossy();
-        if crate_name.starts_with("rustc")
-            || crate_name.starts_with("syntax")
-            || crate_name == "arena"
-            || crate_name == "fmt_macros"
+        if crate_name != "rustc_version"
```

Comment:
Little hack, since `rustc_version` is a external dep and therefore needs to be excluded here. That was the easiest way. :)